test/integration/ci-operator-prowgen/input/config/super/duper/super-duper-release-4.19__periodics.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  base:
+    name: "4.19"
+    namespace: ocp
+    tag: base
+build_root:
+  from_repository: true
+operator:
+  bundles:
+  - as: super-duper-bundle
+    base_index: operator-index
+  substitutions:
+  - pullspec: quay.io/openshift/super-duper:latest
+    with: pipeline:super-duper
+releases:
```

Comment:
Let skip any of the configuration that is not related to the expected generation. That will make the test case to be more descriptive.